### DISCUSSION TITLE
Is it possible to write custom GPU kernels that interface with the U8, b_fs_yx_fsv16 format?

### DISCUSSION THREAD

**Discussion by jonmcwongDR** [openvinotoolkit/openvino#23777] [Q&A]
I didn't quite see any mention of this in the docs, only the ability to read and write different permutations of BFYX. Have I missed something or is this a feature that is planned for the future?

EDIT: as an extension are there any plans to allow optimised models to output this format?

**Comment by rkazants** [comment]
Hi @jonmcwongDR,

Did you check this documentation: https://docs.openvino.ai/2024/documentation/openvino-extensibility/custom-gpu-operations.html?

@p-durandin, @vladimir-paramuzov, please help with more details

Best regards,
Roman

**Reply by jonmcwongDR** [reply]
Hi, the desired behaviour I am looking for is not in this documentation.
